recipes/evo/recipe.yaml

```diff
@@ -0,0 +1,67 @@
+context:
+  version: 1.31.1
+  python_min: 3.10
+
+package:
+  name: evo
+  version: ${{ version }}
+
+source:
+- url: https://files.pythonhosted.org/packages/37/c9/450f89b92ad6fccf66152267dfb52f6bf9e85476b217a14ac4a09a5ccadd/evo-${{ version }}.tar.gz
+  sha256: 81bd49dbdcae94eb30597c21e038be60140f82ae794ec62b04f2cd5a1ed9d8a6
+
+build:
+  noarch: python
+  number: 0
+  script: ${{ PYTHON }} -m pip install . --no-deps --no-build-isolation
+  python:
+    entry_points:
+    - evo  =  evo.main_evo:main
+    - evo_ape  =  evo.entry_points:ape
+    - evo_config  =  evo.main_config:main
+    - evo_fig  =  evo.main_fig:main
+    - evo_ipython  =  evo.main_ipython:main
+    - evo_res  =  evo.entry_points:res
+    - evo_rpe  =  evo.entry_points:rpe
+    - evo_traj  =  evo.entry_points:traj
+
+requirements:
+  host:
+  - python ${{ python_min }}.*
+  - pip
+  - hatchling
+  run:
+  - python >=${{ python_min }}
+  - argcomplete
+  - colorama
+  - matplotlib-base
+  - natsort
+  - numexpr
+  - numpy
+  - pandas
+  - pillow
+  - pygments
+  - pyyaml
+  - rosbags >=0.9.20
+  - scipy
+  - seaborn
+  - contextily
+
+tests:
+- python:
+    imports:
+    - evo
+    # pip check fails as rosbags doesn't include version information
```

Comment:
`rosbags` needs setuptools-scm or an env variable to hardcode the version if just setuptools-scm doesn't fix it. See https://github.com/conda-forge/rosbags-feedstock/blob/main/recipe/meta.yaml

PS: We prefer to fix this now than keep disableing pip-check in everything that depends on `rosbags`.

Reply:
All fixed up, thanks for the pointer to setuptools-scm!